pandas/tests/indexes/multi/test_contains.py

```diff
@@ -58,6 +60,24 @@ def test_isin():
     assert result.dtype == np.bool_
 
 
+@pytest.mark.parametrize('values, expected_badix',
+                         [([('foo',), ('bar', 3), ('quux',)], 0),
+                          ([('foo', 2), ('bar', 3, 2), ('quux', 4)], 1)])
+def test_isin_values_raises(values, expected_badix):
+    idx = MultiIndex.from_arrays([
+        ['qux', 'baz', 'foo', 'bar'],
+        np.arange(4)
+    ])
+
+    nlvl = len(idx.levels)
+    msg = re.escape('Length of each element in values must '
+                    'match number of levels in MultiIndex. '
+                    'len({}) != {}'.format(values[expected_badix], nlvl))
```

Comment:
i would be simpler, and more explicit, if this was replaced with the literal string raised by the test.

```suggestion
                    + expected)
```

Reply:
I like that

Reply:
done